torch/ao/quantization/fx/prepare.py

```diff
@@ -433,7 +435,13 @@ def get_arg_target_dtype_as_output(
     argument in quantized graph will match what is specified by the qconfig
     """
     assert isinstance(arg, Node)
-    if is_activation_post_process_node(arg, modules):
+    # Custom module LSTM output is a tuple that we broke down into the internal nodes in order
```

Comment:
this makes sense if I read the PR summary, but not by itself.  Can we add some more context here?

Reply:
Ok, will do